utils.py

```diff
@@ -112,21 +119,21 @@ def get_closest_strike_price(ticker: str, option_type: OptionType) -> Decimal:
     logger.info(
         f"Current Stock price: {price} \nFinding closest out-of-the-money strike price..."
     )
-    
+
     if r.options.find_options_by_strike(
         ticker, nearest_half_dollar_increment, option_type, info="expiration_date"
     ):
         logger.info(f"Closest strike price: {nearest_half_dollar_increment}")
         return nearest_half_dollar_increment
-    
+
     for i in range(4):
-        if option_type == 'call':
+        if option_type == "call":
             nearest_half_dollar_increment = nearest_half_dollar_increment + Decimal(0.5)
-        if option_type == 'put':
+        if option_type == "put":
```

Comment:
Might as well throw `OptionType.call` and `OptionType.put` in there too

I'm on my phone's GitHub so I can't edit the code rn